### ISSUE TITLE
Lupus nephritis

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6831]
**Preferred term label:**
Lupus nephritis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lupus nephritis is a glomerulonephritis and constitutes one of the most severe organ manifestations of systemic lupus erythematosus. Lupus nephritis is subclassified in six distinct classes, that represent different manifestations and severities of renal involvement and guide the therapeutic management.
PMID: 31974366

**Parent term (use hpo.jax.org/app)**
Glomerulonephritis HP:0000099

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM #152700, ORPHA:536

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**General comment to this new term and the following subterms**
Although all of the different subclasses of Lupus nephritis can in principal be characterized by existing terms, they are currently not linked to Systemic lupus erythematosus HP:0002725 (e.g. Mesangial immune complex deposition HP:0033295, Subendothelial glomerular basement membrane electron dense deposits HP:0033500). To me it would make sense to have the subforms of Lupus nephritis and its subclassifications as individual terms, which can then again be linked to the individual pathological/morphological terms. Class III and IV have more subforms, which I would define, if you agree with the need for terms for lupus nephritis.

**Comment by pnrobinson** [comment]
added
